CHANGES_NEXT_RELEASE

```diff
@@ -39,6 +43,7 @@
 - Fix: Correct error returned when trying to replace a non-existing attribute in PUT /v2/entities/<entity-id>/attrs/<attr-name> (Issue #2221)
 - Fix: Empty string detection for entity id, entity type and attribute name in NGSIv2 request URLs (#1426, #1487)
 - Fix: Trailing slashes in URI path no longer silently removed, but considered and giving adequate error responses
+- Fix: Initial notifications were counted twice (which gave erroneous timesSent in GET /v2/subscriptions/<subId>)
```

Comment:
Under which circustances? Only in the case of using case? In that case, the line should be adjusted:

Fix: Initial notifications were counted twice when csub cache was used (which gave erroneous timesSent in GET /v2/subscriptions/<subId>)